src/defaultTheme.js

```diff
@@ -0,0 +1,144 @@
+const baseConfig = {
+  colors: {
```

Comment:
We should add a "type" either "light" or "dark" alongside "colors", "typography" etc.